src/content/docs/start/Frontend Configuration/sveltekit.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Svelte
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+SvelteKit is a meta-framework for Svelte. Learn more about SvelteKit at https://kit.svelte.dev/. This guide is accurate as of SvelteKit 2.5.7 / Svelte 4.2.15.
+
+## Checklist
+
+- Use [SGG](https://kit.svelte.dev/docs/adapter-static) and/or [SPA](https://kit.svelte.dev/docs/single-page-apps) via `static-adapter`. Tauri doesn't support server-based solutions.
+- Use `build/` as `frontendDist` in `tauri.conf.json`.
+
+## Example Configuration
+
+1. Install `@sveltejs/adapter-static`:
+
+<CommandTabs
+	npm="npm install --save-dev @sveltejs/adapter-static"
+	yarn="yarn add -D @sveltejs/adapter-static"
+	pnpm="pnpm add -D @sveltejs/adapter-static"
+/>
+
+2. Update Tauri configuration:
+
+<Tabs>
+	<TabItem label="npm">
+
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "npm run dev",
+		"beforeBuildCommand": "npm run build",
+		"devPath": "http://localhost:5173",
+		"distDir": "../build"
+	}
+}
+```
+
+</TabItem>
+<TabItem label="yarn">
+
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "yarn dev",
+		"beforeBuildCommand": "yarn build",
+		"devPath": "http://localhost:5173",
+		"distDir": "../build"
+	}
+}
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "pnpm dev",
+		"beforeBuildCommand": "pnpm build",
+		"devPath": "http://localhost:5173",
+		"distDir": "../build"
+	}
+}
+```
+
+</TabItem>
+</Tabs>
+
+3. Update SvelteKit configuration:
+
+{/* TODO: a highlighting feature would be nice here. Only the import changed slightly which is hard to spot. */}
+
+```js
+// svelte.config.js
```

Comment:
```suggestion
```js title="svelte.config.js" {1}
```
for title either a comment `//` or `title` works. But with comment, the highlighted line would be {2}, generally not a problem, just like to avoid

Reply:
cool thanks